xemutest/test_base.py

```diff
@@ -20,6 +20,33 @@
 log = logging.getLogger(__file__)
 
 
+class TestEnvironment:
+	"""Encapsulates environment information needed to run the tests."""
+	def __init__(
+			self,
+			private_path: str,
+			xemu_path: Optional[str],
+			ffmpeg_path: Optional[str],
+			disable_fullscreen: bool = False):
+		self.private_path = private_path
+
+		cur_dir = os.getcwd()
+		if not xemu_path:
+			if platform.system() == 'Windows':
+				self.xemu_path = os.path.join(cur_dir, 'xemu.exe')
+			else:
+				self.xemu_path = 'xemu'
+		else:
+			self.xemu_path = xemu_path
+
+		self.ffmpeg_path = ffmpeg_path
```

Comment:
If `ffmpeg_path` isn't provided, let's use `ffmpeg[.exe]` (resolved by environment PATH)

Reply:
nvm, I see you changed it below